src/libopensc/cards.h

```diff
@@ -204,6 +204,7 @@ enum {
 	SC_CARD_TYPE_IASECC_BASE = 25000,
 	SC_CARD_TYPE_IASECC_GEMALTO,
 	SC_CARD_TYPE_IASECC_OBERTHUR,
+	SC_CARD_TYPE_IASECC_LATVIA_EID,
```

Comment:
add your card type to the end of oberthur cards to preserve the existing ids

Reply:
Sorry, not sure if I understand correctly. Should I move new type after  `SC_CARD_TYPE_IASECC_MI2`?

Reply:
yes, exactly